docs/designers-developers/developers/tutorials/create-block/readme.md

```diff
@@ -0,0 +1,31 @@
+
+# Create Block Tutorial
+
+The goal of this tutorial is to get you started creating your first block for the WordPress Block Editor. We will create a simple block that allows the user to type a message and styles it.
+
+The tutorial includes setting up your development environment, tools, and getting comfortable with the new development model. If you are already comfortable, try the quick start below, otherwise step through whatever part of the tutorial you need.
+
+## Quick Start
+
+The `@wordpress/create-block` package exists to create the necessary block scaffolding to get you started. See [create-block package documentation](https://www.npmjs.com/package/@wordpress/create-block) for additional features. This quick start assumes you have a development environment with node installed, and a WordPress site. From your plugins directory, to create your block run:
+
+```bash
+npx @wordpress/create-block starter-block
```

Comment:
I always use `npm init @wordpress/block starter-block`. Both are valid and will work exactly the same 😃 I guess my preference is based on what projects like `create-react-app` promote in their documentation.

Reply:
I prefer the npx in this case because we are talking about the `@wordpress/create-block` package but then the command refers to a different package name.

My guess is the older syntax is from before `npx` was common and bundled with `npm` and sticks around since people are familiar with it still.